src/warnet/warnet.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, config_dir):
         self.bitcoin_network: str = "regtest"
         self.docker_network: str = "warnet"
         self.subnet: str = "100.0.0.0/8"
-        self.graph = None
+        self.graph: Optional[networkx.Graph] = None
```

Comment:
Is this unrelated?

Reply:
Yup. I just fix the typechecker errors when I'm near by :)